bim2sim/assets/enrichment/usage/UseConditions.json

```diff
@@ -17,7 +17,7 @@
         "lighting_efficiency_lumen": "TEASER-v1.0.1",
         "ratio_conv_rad_lighting": "TEASER-v1.0.1",
         "use_constant_infiltration": "TEASER-v1.0.1",
-        "infiltration_rate": "TEASER-v1.0.1",
+        "base_infiltration": "TEASER-v1.0.1",
```

Comment:
Please update the teaser version number to latest version, in all use conditions templates, either for all parameters (if still valid), or just for base_infiltration. Should be changed also in test resources repository

Reply:
adjusted with cc5131e3